Makefile

```diff
@@ -177,7 +177,7 @@ test-unstable:
 
 .PHONY: test-fullstack
 test-fullstack:
-	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=30 RETRY=15 PROVE_ARGS="$$HARNESS t/full-stack.t"
+	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=300000 RETRY=50 STABILITY_TEST=1 PROVE_ARGS="$$HARNESS t/full-stack.t"
```

Comment:
Adding `RETRY=50 STABILITY_TEST=1` here since the extra PR was making it more confusing

Reply:
Well, it seems virtually impossible to get 50 runs without unrelated errors...